src/main/java/com/philderbeast/prisonpicks/Events.java

```diff
@@ -56,43 +65,33 @@ public void onBlockBreak(BlockBreakEvent event)
         }
 
         aoepick = false;
+
     }
+    
 
-    @EventHandler
-    public void onPlayerInteract(PlayerInteractEvent event)
-    {
-        Player player = event.getPlayer();
-        ItemStack item = player.getInventory().getItemInMainHand();
 
         //are they using one of our picks
+        ItemMeta itemMeta = item.getItemMeta();
         if ((event.getAction() == Action.RIGHT_CLICK_AIR || event.getAction() == Action.RIGHT_CLICK_BLOCK)
             && (Xpick.isPick(item) || Pickoplenty.isPick(item) || XPickoPlenty.isPick(item))
-            && item.getDurability() > 0)
+            && ((Damageable) itemMeta).getDamage() > 0)
         {
             if (!hideRepair.contains(player.getName()))
             {
                 if (Pickoplenty.isPick(item))
                 {
-                    player.sendMessage(Config.CHAT_POP_REPAIR + "[Pickaxe Repaired]");
+                    player.spigot().sendMessage(ChatMessageType.ACTION_BAR, TextComponent.fromLegacyText(Config.CHAT_POP_REPAIR + "[Pickaxe Repaired]"));
                 } else if (XPickoPlenty.isPick(item))
                 {
-                    player.sendMessage(Config.CHAT_XPOP_REPAIR + "[Pickaxe Repaired]");
-                } else
+                    player.spigot().sendMessage(ChatMessageType.ACTION_BAR, TextComponent.fromLegacyText(Config.CHAT_XPOP_REPAIR + "[Pickaxe Repaired]"));
+                }else
                 {
-                    player.sendMessage(Config.CHAT_EXPLOSIVE_REPAIR + "[Pickaxe Repaired]");
+                    player.spigot().sendMessage(ChatMessageType.ACTION_BAR, TextComponent.fromLegacyText(Config.CHAT_EXPLOSIVE_REPAIR + "[Pickaxe Repaired]"));
                 }
             }
-            short s = 0;
-            item.setDurability(s);
+            //short s = 0;
+            
+            ((Damageable) itemMeta).setDamage(0);
+            item.setItemMeta(itemMeta);
         }
     }
-
-    @EventHandler
-    public void onBlockPlace(BlockPlaceEvent event)
```

Comment:
why is this removed? its stops somone placing a high value block to pop the blocks around it.

Reply:
mistakenly removed it